_search-plugins/star-tree-index.md

```diff
@@ -156,12 +170,12 @@ The following metric aggregations are supported as of OpenSearch 2.18:
 - [Value count]({{site.url}}{{site.baseurl}}/aggregations/metric/value-count/)
 - [Average]({{site.url}}{{site.baseurl}}/aggregations/metric/average/)
 
-To use aggregations:
+To use searchable aggregations with a star tree index, remember the following prerequisites:
 
 - The fields must be present in the `metrics` section of the star-tree configuration.
 - The metric aggregation type must be part of the `stats` parameter.
 
-### Aggregation example
+##### Example
```

Comment:
@Naarcha-AWS Would we normally use an L5 heading?

Reply:
Not typically. I'll go ahead and remove the header.

Reply:
Why not just have it be an L4?

Reply:
The example is related to metric aggregations specifically. Not having an L4 simplifies the formatting. Furthermore, line 202 should be an L4 since its intended as a subsection "Supported aggregations".